modules/debts_api/app/models/debts_api/v0/form5655_submission.rb

```diff
@@ -84,6 +84,7 @@ def register_failure(message)
       update(error_message: message)
       Rails.logger.error('Form5655Submission failed', message)
       StatsD.increment("#{STATS_KEY}.failure")
+      StatsD.increment('silent_failure', [service: 'debt-resolution', function: 'register_failure'])
```

Comment:
The tags need to be an array of strings 

```suggestion
      StatsD.increment('silent_failure', tags: ['service:debt-resolution', 'function:register_failure'])
```

Reply:
Ah cool. Thank you for the suggestion.